builds/android/app/src/gamebrowser/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Was just thinking we could add this to the main CMakeLists as well.
However, it is currently really convoluted, so maybe we should split other things out like this.

Reply:
Imo it would make sense to move all the system dependent stuff in CmakeLists.txt in the build/ folder. Currently the main file is quite messy.

When devkitXXX and VitaSDK are ever built through CMake we could hide all the custom stuff (build_vpk, make_dol, etc.) in subfolders.

Reply:
> When devkitXXX and VitaSDK are ever built through CMake we could hide all the custom stuff (build_vpk, make_dol, etc.) in subfolders.

of course!